addon/components/o-s-s/smart/immersive/logo.stories.js

```diff
@@ -0,0 +1,115 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { action } from '@storybook/addon-actions';
+
+export default {
+  title: 'Components/Smart::Immersive::Logo',
+  component: 'oss-smart-immersive-logo',
+  argTypes: {
+    icon: {
+      description: 'Icon name in format "name:color", e.g. "rabbit:orange"',
```

Comment:
icons & colors are defined here -> https://github.com/upfluence/upfluence-web/blob/staging/packages/publishr-admin-web/addon/models/campaign.ts#L61

I think we should store them somewhere in the project in a dedicated file (and not use the "campaign" prefix ofc)

This way we can:

use them in storybook with the proper values
Ensure the parameters passed to the component are valid (for both colors & icons)
have loops of tests over them